frontend/packages/console-app/src/components/pdb/PDBFormPage.tsx

```diff
@@ -69,17 +69,16 @@ export const PDBFormPage: React.FC<{
   const k8sObj = pdbToK8sResource(initialPDB);
 
   const YAMLEditor: React.FC<YAMLEditorProps> = ({ onChange, initialYAML = '' }) => {
```

Comment:
> lets remove the `initialYAML` since its not needed anymore. Also in the `YAMLEditorProps`

@jhadvig The `initialYAML` is needed in the `mergeInitialWithExistingData` below.

Reply:
oh, sorry ...my bad